languages/python/oso/polar/query.py

```diff
@@ -99,9 +94,15 @@ def handle_external_call(self, data):
                 self.ffi_query.application_error(str(e))
                 self.ffi_query.call_result(call_id, None)
                 return
-
-            if callable(attr):  # If it's a function, call it with the args.
+            if (
+                callable(attr) and not data["args"] is None
+            ):  # If it's a function, call it with the args.
+                args = [self.host.to_python(arg) for arg in data["args"]]
                 result = attr(*args)
+            elif not data["args"] is None:
+                raise RuntimeError(
+                    f"tried to call '{attribute}' but it is not callable"
+                )
```

Comment:
nit: would prefer this as a new exception type that can be caught by users of the library

Reply:
This is consistent with how other exceptions are done in the Python library. Don't want to solve here.